conf/distro/include/raspberrypi_generic_dunfell.conf

```diff
@@ -0,0 +1,18 @@
+PREFERRED_VERSION_cmake = "3.16.%"
+PREFERRED_VERSION_cmake-native = "3.16.%"
+PREFERRED_VERSION_gstreamer1.0 = "1.16.%"
+PREFERRED_VERSION_gstreamer1.0-libav = "1.16.%"
+PREFERRED_VERSION_gstreamer1.0-plugins-bad = "1.16.%"
+PREFERRED_VERSION_gstreamer1.0-plugins-base = "1.16.%"
+PREFERRED_VERSION_gstreamer1.0-plugins-good = "1.16.%"
+PREFERRED_VERSION_gstreamer1.0-plugins-ugly = "1.16.%"
+PREFERRED_VERSION_gstreamer1.0-omx = "1.16.%"
+PREFERRED_VERSION_linux-raspberrypi = "4.19.%"
+
+BBMASK_append = " meta-wpe/recipes-core/glibc/glibc-locale_2.24.bbappend"
```

Comment:
Why do we need BBMASKs? Can't we just use PREFERRED_VERSION?

Reply:
We cannot remove all BBMAKS, since some bbappend is not available in all bistro, ins such case we have to keep it